README.md

```diff
@@ -113,9 +115,9 @@ of all panels on which they have previously been seen. A gene will be treated as
 before, or features in a phenotype-matched panel on which it has not been seen before. i.e.
 
 - if a Gene is promoted to `Green` on the Mendeliome, it will be recorded as `New`, and the prior data will be extended
-to show that the gene was seen on the Mendeliome.
+  to show that the gene was seen on the Mendeliome.
```

Comment:
IMO, just make this one line, and use soft-wrapping in VSCode:

```jsonc
// in the settings.json
"[markdown]": {
    // ... other settings
    "editor.wordWrap": "wordWrapColumn",
    "editor.wrappingIndent": "same",
    "editor.wordWrapColumn": 80,
},
```

Reply:
This one was caused by markdownlint... or ruff. One or the other. On-save this got split off back onto separate lines. I've reformatted all markdown files in the repo to remove artificial line breaks